CONTRIBUTING.md

```diff
@@ -4,34 +4,34 @@ Please follow common guidelines for our projects [here](https://github.com/packi
 
 ## Reporting Bugs
 
-- [List of known issues](https://github.com/packit/packit-service/issues) and in case you need to create a new issue, you can do so [here](https://github.com/packit/packit-service/issues/new).
+- [List of known issues](https://github.com/packit/packit-service/issues); if you need to create a new issue, you can do so [here](https://github.com/packit/packit-service/issues/new).
 - Getting a version of `packit`:<br>
   `rpm -q packit` or `pip3 freeze | grep packitos`
 
 ## Building images locally
 
-For building images SOURCE_BRANCH build arg is required. It allows select which branch should be
-used during automated build and test process, based on deployment stage it was executed from.
+For building images the SOURCE_BRANCH build arg is required. It allows selecting which branch should be
+used during the automated build and test process, based on the deployment stage it was executed from.
 
-Docker file build arg SOURCE_BRANCH is set to:
+The Podman file build arg SOURCE_BRANCH is set to:
```

Comment:
```suggestion
In the Dockerfiles SOURCE_BRANCH is set to:
```

As there are 3 Dockerfiles in this repo, and there are called `Dockerfile*`, so there removing the "docker" terminology here would create confusion. The right term and name for these files would be "Containerfile".

Reply:
Made the suggested change.